proxy.py

```diff
@@ -229,36 +239,60 @@ def check_convergence(self):
                     self.epochs + 1, min(self.err_te_hist)
                 )
             )
-      
+
     @abstractmethod
     def evaluate(self, data):
         self.model.eval()
         with torch.no_grad():
             output = self.model(data).cpu().detach().numpy()
-            return output  
+            return output
 
     @abstractmethod
     def base2proxy(self, state):
```

Comment:
Let's remove base2proxy as well (the preprocess method that replaces it is in the model(nn.Module) class directly and not the child Proxy class)